cmd/client_test.go

```diff
@@ -111,3 +111,59 @@ func TestClient_CreateJobSpec(t *testing.T) {
 		assert.Equal(t, test.nJobs, len(numberOfJobs))
 	}
 }
+
+func TestClient_CreateJobRun(t *testing.T) {
+	t.Parallel()
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+	client, _ := cltest.NewClientAndRenderer(app.Store.Config)
+
+	tests := []struct {
+		name    string
+		json    string
+		jobType int
+		badID   bool
+		errored bool
+	}{
+		{"Create_Success", `{"value": 100}`, 0, false, false},
```

Comment:
Rather than passing a `jobType`, just place the job in the table:

```
	tests := []struct {
		name    string
		json    string
		job models.Job
		badID   bool
		errored bool
	}{
{"CreateSuccess", `{"value": 100}`, cltest.NewJobWithWebInitiator(), false, false},
```

Also, remove the underscore since the other names are camel cased.